sympy/assumptions/tests/test_query.py

```diff
@@ -2409,6 +2409,9 @@ def test_relational():
 
 
 def test_issue_25221():
-    assert ask(Q.transcendental(x), Q.algebraic(x) | Q.positive(y,y)) is None
+    assert ask(Q.transcendental(x), Q.algebraic(x) | Q.positive(y)) is None
     assert ask(Q.transcendental(x), Q.algebraic(x) | (0 > y)) is None
     assert ask(Q.transcendental(x), Q.algebraic(x) | Q.gt(0,y)) is None
+
+def test_issue_25276():
+    raises(TypeError, lambda: Q.positive(x,y))
```

Comment:
It might be a good idea to add cases for all the predicate in case someone gets rid of the _arity attribute from one of the classes. Also maybe add a short comment here explaining what this is testing for.